src/Components/Facility/FacilityHome.tsx

```diff
@@ -75,6 +77,27 @@ export const FacilityHome = ({ facilityId }: Props) => {
     },
   });
 
+  const fetchImage = async () => {
+    try {
+      let imageUrl = facilityData?.read_cover_image_url;
+      if (imageUrl) {
+        await fetch(`${imageUrl}`, {
+          headers: { "Force-Revalidate": "1" },
+          credentials: "include",
+          mode: "no-cors",
+        });
+        imageUrl += "?" + Date.now();
+        setCoverImageUrl(imageUrl);
```

Comment:
Let's not add query params.

Reply:
Will remove 👍  I added query params since fetch/vary header behavior wasn't consistent on Firefox (though it's fine on Safari).